src/OSPSuite.Core/Services/PredictedVsObservedChartService.cs

```diff
@@ -88,39 +88,47 @@ private DataColumn createIdentityColumn(IDimension mergedDimension, params float
          return values;
       }
 
-      public void AddCurvesFor(IEnumerable<DataColumn> observationColumns, DataColumn calculationColumn, ParameterIdentificationPredictedVsObservedChart chart, Action<DataColumn, Curve> action = null)
+      public void AddCurvesFor(IEnumerable<DataColumn> observationColumns, DataColumn calculationColumn, PredictedVsObservedChart chart,
+         Action<DataColumn, Curve> action = null)
       {
          observationColumns.Each(observationColumn => plotPredictedVsObserved(observationColumn, calculationColumn, chart, action));
       }
 
-      public void SetXAxisDimension(IEnumerable<DataColumn> observationColumns, ParameterIdentificationPredictedVsObservedChart chart)
+      public void ConfigureAxesDimensionAndTitle(IEnumerable<DataColumn> observationColumns, PredictedVsObservedChart chart)
       {
-         var dataColumns = observationColumns.ToList();
-
-         var defaultDimension = mostFrequentDimension(dataColumns);
          var xAxis = chart.AxisBy(AxisTypes.X);
+         var yAxis = chart.AxisBy(AxisTypes.Y);
 
-         if (defaultDimension != null)
-            xAxis.Dimension = defaultDimension;
-
-         xAxis.Scaling = chart.AxisBy(AxisTypes.Y).Scaling;
-         xAxis.UnitName = chart.AxisBy(AxisTypes.Y).UnitName;
+         var defaultDimension = mostFrequentDimension(observationColumns.ToList());
 
+         setAxisDimension(defaultDimension, chart, xAxis);
```

Comment:
@georgeDaskalakis what are those changes about here? Is this to fix another issue?

Reply:
@msevestre yes this is  a bugfix here